nix/modules/tezos-accuser.nix

```diff
@@ -0,0 +1,70 @@
+# SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+{config, lib, pkgs, ...}:
+
+with lib;
+
+let
+  tezos-accuser-pkgs = {
+    "007-PsDELPH1" =
+      "${pkgs.tezos.tezos-accuser-007-PsDELPH1}/bin/tezos-accuser-007-PsDELPH1";
+    "008-PtEdo2Zk" =
+      "${pkgs.tezos.tezos-accuser-008-PtEdo2Zk}/bin/tezos-accuser-008-PtEdo2Zk";
+  };
+  cfg = config.services.tezos-accuser;
+in {
+  options.services.tezos-accuser = rec {
```

Comment:
Does this really need to be `rec`? (Asking because recursive definitions of options are always sus.)